chapeau.py

```diff
@@ -122,6 +120,52 @@ def parse_function(data):
 
 routing_dictionary = {}
 
+def add_route(path, handler):
+	'''
+		params:
+			path:   <str>
+			handler: <function>
+
+		Adds the path as a key to the routing dictionary and maps
+		it to the function assigned to the path 
+	'''
+	if not isinstance(path, str):
+		raise TypeError
+	routing_dictionary[path] = handler
+
+def route(path):
+	'''
+		Decorator function
+		params:
+			path: <str>
+		Example:
+			@route('/user/leta')
+			def user_leta(request, client):
+				chapeau.render(client, '/path/to/file', *args, **kwargs)
+	'''
+	def route_decorator(func):
+		add_route(path, func)
+		return func(*args, **kwargs)
```

Comment:
This line will give us an error because `args` and `kwargs` are not defined.  That's easy to fix though, because we actually don't even need to invoke `func` here.  Instead, we can just return the object `func`.  

Note:  Often decorators *do* return an invocation of the original function, but the reason for this behaviour is so that additional functionality occurs when the decorated function is called.  Since we don't need any additional behaviour to happen when our decorated functions are called, this step is unnecessary (and more work).  Had we wanted to add additional functionality to our decorated functions, then we would need to have an additional layer of nestedness (similar to what you had previously) to actually take in the functions arguments and call the function with those arguments.